js/components/portal-dashboard/student-names.tsx

```diff
@@ -12,19 +14,30 @@ export class StudentNames extends React.PureComponent<IProps> {
   render() {
     const { students, isAnonymous, isCompact } = this.props;
     const compactClass = isCompact ? css.compact : "";
+    let rowClass = css.studentNameRow;
```

Comment:
use const here.

Reply:
Deleted the line and where it's used instead.